### PR TITLE
Added Demo Documentation

### PR DESCRIPTION
**Pull request by Satgoy152**

### CODE REVIEW

**Comment by Satgoy152** [comment]
worked under @vikas84bf

File: models/face_detection_yunet/README.md [hunk 1]
```diff
@@ -19,13 +20,81 @@ Results of accuracy evaluation with [tools/eval](../../tools/eval).
...
+Command: `--save (Boolean True or False)`
+
+Usage: Set “True” to save file with results (i.e. bounding box, confidence level). Invalid in case of camera input. Default will be set to “False”.
+
+#### Show Visualization Window
+
+Command: `--vis (Boolean True or False)`
+
+Usage: Default will be set to “True” and will open a new window to show results. Set to “False” to stop visualizations from being shown. Invalid in case of camera input.
+
```
Comment (fengyuentau) [inline models/face_detection_yunet/README.md:97]: These documentations should be the help messages of arguments as follows:

https://github.com/opencv/opencv_zoo/blob/1ecf13e1deb97c9ae6a715ceb35553fb5eb37304/models/face_detection_yunet/demo.py#L34-L44

These arguments may be changed from time to time. If the documentation to these arguments are put in readme,  we will have to update readme if arguments are updated, which does not make sense to me. From my perspective, It is better to show the documentation to these arguments with `python demo.py --help`, which you can put along with the demo commands:
```.diff
# detect on camera input
python demo.py
# detect on an image
python demo.py --input /path/to/image
+ # show help messages
+ python demo.py --help
```

**Review by fengyuentau — CHANGES_REQUESTED** [review]
Thank you for the contribution! Please take a look at my comment below.

File: models/face_detection_yunet/README.md (hunk 1 above)
Comment (Satgoy152) [reply models/face_detection_yunet/README.md:97]: Yes, that makes sense. Would you like me to make those changes and create a new commit?

Comment (fengyuentau) [reply models/face_detection_yunet/README.md:97]: Yes, please do so.

**Comment by Satgoy152** [comment]
I updated the help messages and removed them from all the read-me's. Please take look.

File: models/face_detection_yunet/demo.py [hunk 2]
```diff
@@ -32,15 +32,15 @@ def str2bool(v):
     print('This version of OpenCV does not support TIM-VX and NPU. Visit https://gist.github.com/fengyuentau/5a7a5ba36328f2b763aea026c43fa45f for more information.')
```
Comment (fengyuentau) [inline models/face_detection_yunet/demo.py:32]: Could you replace this link (and the links in other demos) with https://github.com/opencv/opencv/wiki/TIM-VX-Backend-For-Running-OpenCV-On-NPU?

**Review by fengyuentau — COMMENTED** [review]
Everything looks good to me. Could you handle one last minor comment? After that, this PR can be merged.

File: models/face_detection_yunet/demo.py (hunk 2 above)
Comment (Satgoy152) [reply models/face_detection_yunet/demo.py]: Pushed again with those changes.

**Review by fengyuentau — APPROVED** [review]
👍 Please squash commits into one before merge:
```
git rebase -i HEAD~3
# set the first commit to pick, and others to squash
git push -f
```

**Review by fengyuentau — APPROVED** [review]
👍 Please squash commits into one before merge:
```shell
git rebase -i HEAD~3
# set the first commit to pick, and others to squash
git push -f
```

**Comment by Satgoy152** [comment]
Just to confirm like this?
`pick 8b40c8d Added Demo Documentation
squash 6316a9a Updated help messages
squash 0ca8b0a Changed exception link`

**Comment by fengyuentau** [comment]
Yes. And you will be asked to write a proper commit message for the squashed commit after you save and exit this instruction window.
________________________________
发件人: Satyam Goyal ***@***.***>
发送时间: Monday, October 31, 2022 2:32:02 PM
收件人: opencv/opencv_zoo ***@***.***>
抄送: Yuantao Feng ***@***.***>; Assign ***@***.***>
主题: Re: [opencv/opencv_zoo] Added Demo Documentation (PR #95)


Just to confirm like this?
pick 8b40c8d Added Demo Documentation squash 6316a9a Updated help messages squash 0ca8b0a Changed exception link

―

**Pull request merged**